04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -436,28 +1238,26 @@
        " 'mydb_host': 'db02.datalab.noao.edu'}"
       ]
      },
-     "execution_count": 11,
+     "execution_count": 23,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "qc.list_profiles(\"default\")"
+    "qc.list_profiles('default')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "So let's try a query against the default profile first - let's get a list of all tables in the default database.  Note that in this case we are accessing the *information_schema* table of the database, this table is not included in the TAP service and so we <b>must</b> use the *sql* argument to talk directly to the database."
+    "So let's try a query against the default profile first - let's get a list of all tables in the default database.  Note that in this case we are accessing the *information_schema* table of the database, this table is not included in the TAP service and so we **must** use the *sql* argument to talk directly to the database."
```

Comment:
This example is last, but says "let's use this first". Suggest to either move this example up, or to remove the 'first'.